### PR TITLE
Add checks for RBI AnnexI(7.1), AnnexI(7.2) and AnnexI(7.3)

### PR DESCRIPTION
**Pull request by Subhajit97**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: query/iam/iam_no_inline_policy.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
+select
```
Comment (cbruno10) [inline query/iam/iam_no_inline_policy.sql:1]: @Subhajit97 Can you please rename this query to `iam_group_no_inline_policies`?

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see comment

File: query/iam/iam_no_inline_policy.sql (hunk 1 above)
Comment (Subhajit97) [reply query/iam/iam_no_inline_policy.sql:1]: @cbruno10 This query checks inline policies for IAM users, groups and roles together.
Should we mention `iam_group` in query fileName, or we can add something that matches all three resource types?

Comment (cbruno10) [reply query/iam/iam_no_inline_policy.sql:1]: @Subhajit97 How about `iam_group_user_role_no_inline_policies.sql`?

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**